docs/ansible_turbo_mode.rst

```diff
@@ -0,0 +1,200 @@
+.. _ansible_turbo_mode:
+
+
+******************
+Ansible Turbo mode
+******************
+
+Following document provides overview of Ansible Turbo mode in ``kubernetes.core`` collection.
+
+.. contents::
+   :local:
+   :depth: 1
+   
+
+Synopsis
+--------
+- A brief introduction about Ansible Turbo mode in ``kuberentes.core`` collection.
+
+
+Requirements
+------------
+
+The below requirements are needed on the host that executes this module.
+
+- The ``cloud.common`` collection (https://github.com/ansible-collections/cloud.common)
+- `AIOHTTP <https://docs.aiohttp.org/en/stable/>`_ - Asynchronous HTTP Client/Server for asyncio and Python.
```

Comment:
Oh, @goneri beat me to it.

Reply:
same for me but the way it is implemented is a bit transparent, we can run with or without TurboModule without any notification because the import just default to the standard AnsibleModule, will be great to have a banner just after the import, something

```
try:    
   from ansible_collections.cloud.common.plugins.module_utils.turbo.module import ( 
        AnsibleTurboModule as AnsibleModule,    )  
    AnsibleModule.collection_name = "kubernetes.core"
   
   ansible.banner("Cloud.common is installed, you are running with TurboMode activated")

except ImportError:    
    from ansible.module_utils.basic import AnsibleModule  # noqa: F401
```